base/actions/actions.py

```diff
@@ -493,6 +546,25 @@ def validate_tracing(
     ) -> Dict[Text, Optional[Text]]:
         return self.validate_generic("tracing", dispatcher, value, self.yes_no_data)
 
+    def get_healthcheck_data(self, tracker: Tracker, risk: Text) -> Dict[Text, Any]:
+        return {
+            "deduplication_id": uuid.uuid4().hex,
+            "msisdn": f'{tracker.sender_id.lstrip("+")}',
```

Comment:
Should this not be `f'+{tracker.sender_id.lstrip("+")}'`? We want to ensure that there's a `+`

Reply:
The test only works without the `+`: https://github.com/praekeltfoundation/healthcheck/blob/develop/tbconnect/test_views.py#L35

Reply:
Oh that's interesting. I would expect it to be fine with the `+27856454612`. I think the issue is that we tell the library to expect South Africa, so `27....` is fine, it can tell that it's za, but if you had `263...`, I think the library tries to make it `+27263...`, and then calls it an invalid phone number.